tests/e2e/env/setup.sh

```diff
@@ -210,6 +210,23 @@ cli wp option set woocommerce_enable_signup_and_login_from_checkout "yes"
 echo "Importing WooCommerce shop pages..."
 cli wp wc --user=admin tool run install_pages
 
+# Start - Workaround for > WC 8.3 compatibility by updating cart & checkout pages to use shortcode.
+# To be removed when WooPayments L-2 support is >= WC 8.3
+INSTALLED_WC_VERSION=$(cli_debug wp plugin get woocommerce --field=version)
+IS_WORKAROUND_REQUIRED=$(cli_debug wp eval "echo version_compare(\"$INSTALLED_WC_VERSION\", \"8.3\", \">=\");")
+
+if [[ "$IS_WORKAROUND_REQUIRED" = "1" ]]; then
+	echo "Updating cart & checkout pages for WC > 8.3 compatibility..."
+	# Get cart & checkout page IDs.
+	CART_PAGE_ID=$(cli_debug wp option get woocommerce_cart_page_id)
+	CHECKOUT_PAGE_ID=$(cli_debug wp option get woocommerce_checkout_page_id)
+
+	# Update cart & checkout pages to use shortcode.
+	cli wp post update "$CART_PAGE_ID" --post_content="<!-- wp:shortcode -->[woocommerce_cart]<!-- /wp:shortcode -->"
```

Comment:
Could we get the shortcode commands from constant variables as well? Makes the final command look much more readable imo.

Reply:
Yeah sure. Will move them to variables

Reply:
Changes added on https://github.com/Automattic/woocommerce-payments/pull/7690/commits/1015f853e409ba6a7609cf15b58dc03a65b884f5